src/test/test_ep_cred_client.c

```diff
@@ -111,6 +114,7 @@ static int sent_count;
 static void
 rpc_handle_reply(const struct crt_cb_info *info)
 {
+	assert(info->cci_rc == 0);
```

Comment:
Should there be a message to provide more info about the assert fail using D_ASSERTF instead?

Reply:
particularly since line 82 uses the macro

Reply:
yeah probably, was being lazy:) will change to D_ASSERTF in all cases